dashboard/app/views/admin_reports/admin_stats.html.haml

```diff
@@ -25,27 +25,17 @@
           Female Students:
           = @stats['born_accounts']['female_students']
           = surround '(', ')' do
-            = number_with_precision(100.0 * @stats['born_accounts']['female_students'].to_f / (@stats['born_accounts']['female_students'].to_f + @stats['born_accounts']['male_students'].to_f), precision: 2)
+            = number_with_precision(100.0 * @stats['born_accounts']['female_students'].to_f / (@stats['born_accounts']['female_students'].to_f + @stats['born_accounts']['male_students'].to_f), precision: 2) + '%'
```

Comment:
Note that fixing involved creating the get_gender_percentage helper method. Not a bad thing. :)